pandas/core/categorical.py

```diff
@@ -302,11 +306,31 @@ def copy(self):
         return Categorical(values=self._codes.copy(),categories=self.categories,
                            name=self.name, ordered=self.ordered, fastpath=True)
 
+    def astype(self, dtype):
+        """ coerce this type to another dtype """
+        if is_categorical_dtype(dtype):
+            return self
+        elif is_object_dtype(dtype):
+            return np.array(self)
```

Comment:
that will cause numpy to error if ``dtype=='category'``; 

or are you just talking about where I have ``is_object_dtype`` (and just make the ``return np.array(self, dtype=dtype)``. That would seem to be ok